CONTRIBUTING.md

```diff
@@ -277,6 +277,24 @@ COVERAGE_PROCESS_START DEFAULT=/coverage.rc
 
 The CI/CD system is based on [Drone platform](https://drone.io/) and the server is  https://ci.vmware.run/. More information is found at our [CI.md](https://github.com/vmware/docker-volume-vsphere/blob/master/CI.md)
 
+## Releases for GA and post GA
+The following is the plan for releases for GA and post GA. Before GA, the release name format remains the same.
```

Comment:
Not sure we should call out GA/Post GA, there will be a question from customer side about GA date. 

/CC @tusharnt